librz/include/rz_type.h

```diff
@@ -85,20 +86,31 @@ typedef struct rz_type_struct_member_t {
 	char *name;
 	RzType *type;
 	size_t offset; // in bytes
-	size_t size; // in bits?
```

Comment:
I was actually wondering, if we switch `offset` measure unit from `bytes` to `bits` we will get the [bitfields support](https://github.com/rizinorg/rizin/issues/1240) for free. @Basstorm @thestr4ng3r @ret2libc @wargio what do you think?

Reply:
yes, but change the definition to ut64, so we are sure that is really an ut64

Reply:
sgtm